src/constants/language.ts

```diff
@@ -1,67 +1,71 @@
 // languages that have translation files in `public/locales`
 export const i18nLanguages = [
-  // 'ar',
-  'da',
-  'en',
-  'en-GB',
-  'en-US',
-  'es',
-  'fr',
-  'fr-FR',
-  'it',
-  'ja',
-  'ms',
-  'nb',
-  'sv',
-  // 'ug',
-  'yue',
-  'zh',
-  'zh-CN',
-  'zh-HK',
-  'zh-TW',
+	// 'ar',
+	'da',
+	'en',
+	'en-GB',
+	'en-US',
+	'es',
+	'fr',
+	'fr-FR',
+	'it',
+	'ja',
+	'ms',
+	'nb',
+	'sv',
+	// 'ug',
+	'yue',
+	'zh',
+	'zh-CN',
+	'zh-HK',
+	'zh-TW',
+	'ro',
```

Comment:
Wrong formatting

Reply:
Sorry for the part with the wrong pr title.